Lib/fontTools/misc/plistlib.py

```diff
@@ -410,6 +423,16 @@ def totree(
 
 
 def fromtree(tree, use_builtin_types=None, dict_type=dict):
+    """Converts an XML tree to a plist structure.
```

Comment:
The inflections of the docstrings in this PR are inconsistent. Is it “Convert” or “Converts”?